### DISCUSSION TITLE
Wrong decimal separator during break

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#24400] [Q&A] [answered]
With the default skin, when the game's language is set to Finnish, the accuracy counter at the top of the screen shows the correct decimal separator (a comma), but the decimal separator in the accuracy counter that shows during a break is wrong (a point).

|English|Finnish|
|-----------|----------|
|![Screenshot from 2023-07-28 17-56-19](https://github.com/ppy/osu/assets/101510334/7ae81335-2de3-4738-8d6e-77010f3a71f0)|![image](https://github.com/ppy/osu/assets/101510334/03661615-32d1-4b5d-8e87-24ea2fd97bc0)|

**Comment by Joehuu** [comment, marked as answer]
https://github.com/ppy/osu/pull/24406
